docs/source/using/troubleshooting.adoc

```diff
@@ -51,21 +51,39 @@ $ systemctl enable virtlogd
 ----
 
 [[domain-minishift-already-exists]]
-=== Error starting the VM: ... operation failed: domain 'minishift' already exists ...
+=== KVM: `Error starting the VM: ...operation failed: domain 'minishift' already exists...`
 
-Check for existing VMs and remove them:
+If you try `minishift start` and hit the above error, ensure that you use `minishift delete` to delete the VMs created earlier by you.
+However if this fails and you wish to completely clean up Minishift and start fresh do the following:
 
+. Check if any existing Minishift VM are running:
++
 ----
 $ sudo virsh list --all
+----
+
+. If any Minishift VM is running, stop it:
++
+----
 $ sudo virsh destroy minishift
+----
+
+. Delete the VM:
++
+----
 $ sudo virsh undefine minishift
 ----
 
-[[troubleshooting-xhyve]]
-== xhyve driver
+. Delete the `.minishift/machines` directory using:
++
+----
```

Comment:
I'd start to consistently mark the type of the source examples, in this case we should add 

'[source,sh]'

(this applies to all commands in this page)